src/yarp.c

```diff
@@ -1530,25 +1554,69 @@ parse_arguments(yp_parser_t *parser, yp_node_t *arguments) {
 // present.
 void
 parse_arguments_list(yp_parser_t *parser, yp_arguments_t *arguments) {
```

Comment:
I'm unsure if we want to make the change to this function as we will be allowing `return(1, 2, 3)`. 

Depending on how we want to parse `return(1)`, we could add a max arguments check to it. Though if this parses the argument to `(1)`, then that is a separate issue.